theano/tensor/nnet/tests/test_nnet.py

```diff
@@ -1699,3 +1701,10 @@ def test_binary_crossentropy_reshape():
         fga = theano.function([a], ga, mode=mode)
         utt.assert_allclose(fga(numpy.array([[[[30.]]]], dtype=config.floatX)),
                             numpy.zeros((1, 1, 1, 1), dtype=config.floatX))
+
+SoftsignTester = makeBroadcastTester(
+    op=softsign,
+    expected=upcast_int8_nfunc(lambda inputs: check_floatX(
+        inputs, inputs/(1.0+numpy.fabs(inputs)))),
+    name='SoftsignTester',
```

Comment:
Here, this is missing which test to run. you should add at least a good parameter. You can check the AddTester example in tensor/tests/test_basic.py.